src/TestFramework/TestFramework/Assertions/Assert.AreEqual.cs

```diff
@@ -80,6 +432,12 @@ public static void AreEqual<T>(T? expected, T? actual, IEqualityComparer<T>? com
     public static void AreEqual<T>(T? expected, T? actual, string? message)
         => AreEqual(expected, actual, null, message, null);
 
+    /// <inheritdoc cref="AreEqual{T}(IEquatable{T}?, IEquatable{T}?, string?)" />
+#pragma warning disable IDE0060 // Remove unused parameter - https://github.com/dotnet/roslyn/issues/76578
+    public static void AreEqual<T>(T? expected, T? actual, [InterpolatedStringHandlerArgument(nameof(expected), nameof(actual))] ref AssertAreEqualInterpolatedStringHandler<T> message)
```

Comment:
I don't know if runtime calls this parameter `message`, I would prefer `handler` or `builder` like in the example.

Reply:
Hmmm. My idea was for existing code written on the form `Assert.AreEqual(expected, actual, message: $"... {...} ...")` to pick up the new overload. But looking at `StringBuilder`, it wasn't done this way. However, for `Debug.Assert`, the BCL reused the same parameter name `message`.

I have no idea if this was a very explicit design design.

Reply:
Ok that works for me